### PR TITLE
Neumorphic design

### PR DESCRIPTION
**Pull request by NateMS**
Neumorphic design for todo file.
Minor changes to html, all css is contained in the `neumorphic.css` file.

Alina Wüthrich (@alinalaura)
Nadim Salloum (@NateMS)

### CODE REVIEW

File: week1/neumorphic.css [hunk 1]
```diff
@@ -1 +1,59 @@
...
+    font-weight: lighter;
+    font-size: 40px;
+    font-variant-caps: all-petite-caps;
+}
+
+main {
+    border-radius: 25px;
+    -webkit-box-shadow: 10px 12px 37px -7px rgba(0, 0, 0, 0.49);
+    -moz-box-shadow: 10px 12px 37px -7px rgba(0, 0, 0, 0.49);
+    box-shadow: 10px 12px 37px -7px rgba(0, 0, 0, 0.49);
```
Comment (Dierk) [inline week1/neumorphic.css]: I would assume that box-shadow is now well-enough supported in all browsers that we do not need to repeat in -webkit -moz. But if you do, better use a css custom property (var) to remove the repetition

File: week1/neumorphic.css [hunk 2]
```diff
@@ -1,47 +1,74 @@
...
+    border: none;
+    box-shadow: 20px 20px 60px #b4c5c9, -20px -20px 60px #f4ffff;
+}
+
+.action-buttons {
+    margin-top: 15px;
+}
+
+.action-buttons button {
+    position: initial !important;
```
Comment (Dierk) [inline week1/neumorphic.css]: why do you need !important ?

File: week1/neumorphic.css [hunk 3]
```diff
@@ -1,47 +1,74 @@
...
+    font-size: 20px;
+    height: 50px;
+}
+
+#plus {
+    width: 50px;
+    height: 50px;
+    padding: 10px;
+    font-size: 35px;
+    line-height: 20px;
```
Comment (Dierk) [inline week1/neumorphic.css]: how does px-units fare on resizing or change of font size?

File: week1/neumorphic2.css [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+*/
+
+/* white version */
+:root {
+    --base-color: #ebecf0;
+    --shadow-color: #BABECC;
+    --highlight-color: white;
+    --danger-color: #ed3134;
+    --plus-color: #007f1f;
```
Comment (Dierk) [inline week1/neumorphic2.css]: nice!

File: week1/neumorphic2.css [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+* {
+    font-family: "Helvetica Neue", sans-serif;
+    color: #9296a2;
+    font-size: 16px;
+}
+
+#todoContainer {
+    display:        grid;
+    grid-column-gap: 0.5em;
+    grid-row-gap: 1em;
```
Comment (Dierk) [inline week1/neumorphic2.css]: good use of em

File: week1/neumorphic.css (hunk 1 above)
Comment (NateMS) [reply week1/neumorphic.css]: yes, this is already outdated and was added in a later version (I am assuming you went through commit by commit)

**Comment by Dierk** [comment]
ok, I finally decided not to merge as-is but to create a new dir with the given content to allow the original and the student submission to sit side-by-side.

**Pull request closed without merging**